addons/website_slides/views/slide_question_views.xml

```diff
@@ -12,7 +12,7 @@
                     <h1>
                         <field name="question" default_focus="1" placeholder="Name"/>
                     </h1>
-                    <field name="answer_ids">
+                    <field name="answer_ids" class="o_question_form_answers">
```

Comment:
Hello,

We don't do custom CSS for form view. You can rely on standard rendering. If there are angry people let them see with framework team how to update form view rendering.